rust/core-lib/assets/client_example.toml

```diff
@@ -36,3 +36,6 @@ wrap_width = 0
 
 # If true, wraps lines at the edge of the view. Overrides 'wrap_width'.
 word_wrap = false
+
+# Detect tab and newline settings on file open
```

Comment:
I wonder if these might be two separate settings? 🤷‍♂️

Reply:
It seems the safer choice, I think the typical user of Xi editor would rather complain about too little control than too much...